--- mousepad-0.6.4/org.xfce.mousepad.desktop.in~	2025-03-25 10:39:10.000000000 +0000
+++ mousepad-0.6.4/org.xfce.mousepad.desktop.in	2025-03-25 11:38:43.039438967 +0000
@@ -8,7 +8,7 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=Utility;TextEditor;GTK;
+Categories=X-XFCE;Utility;TextEditor;GTK;
 MimeType=text/plain;application/x-zerosize;
 Actions=preferences;
 Keywords=Edit;Text;Simple;XFCE;Lightweight;Fast;
