diff -urN mousepad-0.2.16/mousepad.desktop.in mousepad-0.2.16.new/mousepad.desktop.in
--- mousepad-0.2.16/mousepad.desktop.in	2008-06-27 22:01:33.000000000 +0200
+++ mousepad-0.2.16.new/mousepad.desktop.in	2009-02-27 23:23:30.000000000 +0100
@@ -1,7 +1,6 @@
 [Desktop Entry]
 Version=1.0
 Type=Application
-Encoding=UTF-8
 _Name=Mousepad
 _Comment=Simple text editor
 _GenericName=Text Editor
@@ -9,5 +8,5 @@
 Icon=mousepad
 StartupNotify=true
 Terminal=false
-MimeType=text/plain
-Categories=Application;Utility;TextEditor;GTK;
+MimeType=text/plain;
+Categories=X-XFCE;Utility;TextEditor;GTK;
