--- mousepad-0.6.3/org.xfce.mousepad.desktop.in.in~	2024-10-14 18:14:42.000000000 +0000
+++ mousepad-0.6.3/org.xfce.mousepad.desktop.in.in	2024-11-07 00:31:27.162207596 +0000
@@ -8,7 +8,7 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=Utility;TextEditor;GTK;
+Categories=X-XFCE;Utility;TextEditor;GTK;
 MimeType=text/plain;application/x-zerosize;
 Actions=preferences;
 Keywords=Edit;Text;Simple;XFCE;Lightweight;Fast;
